vaadin-notification-flow-parent/vaadin-notification-flow/src/test/java/com/vaadin/flow/component/notification/NotificationTest.java

```diff
@@ -135,6 +135,23 @@ public void constructorsWithNoDurationCreateNotCloseableNotifications() {
         }
     }
 
+    @Test
+    public void createDefaultNotification_templateWarningSuppressed() {
+        Notification notification = new Notification();
+
+        Assert.assertTrue("Template warning is not suppressed", notification
+                .getElement().hasAttribute("suppress-template-warning"));
+    }
+
+    @Test
+    public void createNotification_templateWarningSuppressed() {
+        Notification notification = new Notification("test", 0,
+                Notification.Position.MIDDLE);
+
+        Assert.assertTrue("Template warning is not suppressed", notification
```

Comment:
I'm not sure if this test is necessary... ideally, I must have tested all the variants of creating a notification since the Notification constructor is overloaded...

Reply:
One test for one constructor should be enough

Reply:
Left only one test.